data-serving/scripts/setup-db/src/index.ts

```diff
@@ -30,31 +33,47 @@ const setupDatabase = async ({
         const database = await connection.getDB(databaseName);
         print(`Connected to database "${database}"`);
 
-        // If the collection already exists, drop it so we can recreate it
-        // fresh.
+        // If the collection already exists, drop all imported data from it and
+        // apply the latest schema.
+        let collection;
         if (
             (await database.getCollectionNames()).some(
                 (c) => c == collectionName,
             )
         ) {
-            await (await database.getCollection(collectionName)).drop();
-            print('Dropped existing collection');
-        }
-
-        await database.createCollection(collectionName, {
-            validator: schema,
-        });
-        print(`Created collection "${collectionName}" with schema`);
+            collection = await database.getCollection(collectionName);
+            const query = deleteAllDocuments
+                ? {}
+                : { importedCase: { $exists: true, $ne: null } };
+            const results = await collection.remove(query);
+            print(
+                `Dropped ${
+                    deleteAllDocuments ? 'all' : 'imported'
+                } documents (${results.nRemoved} total) 🗎`,
```

Comment:
I am missing that emoji :(
![image](https://user-images.githubusercontent.com/1255432/86216251-0b6a2300-bb7e-11ea-99fc-86c914e12780.png)

Reply:
EEK! it was not a helpful one anyway, hopefully you can see this awesome trash can